### DISCUSSION TITLE
File scope

### DISCUSSION THREAD

**Discussion by chrisstuartparry** [microsoft/vscode-mypy#83] [Q&A] [answered]
Before I switch to using this extension instead of the Mypy VSCode extension by Matan Gover, I wanted to check what this extension can check; namely, does it run on the entire workspace, or just each file separately? Apologies if this is an obvious answer, I'm not too familiar with the inner workings of linters and whatnot.

Thank you!

**Comment by karthiknadig** [comment, marked as answer]
We plan on adding a mode to check entire workspace: https://github.com/microsoft/vscode-mypy/issues/81

It handles `notes` from `mypy` better. Eventually we plan on adding type hints from `mypy`

**Reply by chrisstuartparry** [reply]
Ah that's great to see, thank you!

Also, sorry, which extension handles `notes` better? (I also don't know what `notes` are in mypy, but I'll go google it).
